axelrod/strategies/better_and_better.py

```diff
@@ -0,0 +1,29 @@
+from axelrod import Actions, Player, random_choice
+
+C, D = Actions.C, Actions.D
+
+class BetterAndBetter(Player):
+    """
+    Defects with probability of '(1000 - current turn) / 1000'.
+    Therefore it is less and less likely to defect as the round goes on.
+
+    Names:
+        - Better and Better: [PRISON1998]_
+
+    """
+
+    name = 'Better and Better'
+    classifier = {
+        'memory_depth': float('inf'),
+        'stochastic': True,
+        'makes_use_of': set(),
+        'long_run_time': False,
+        'inspects_source': False,
+        'manipulates_source': False,
+        'manipulates_state': False
+    }
+
+    def strategy(self, opponent):
+        current_round = len(self.history) + 1
+        probability = float(current_round) / 1000
```

Comment:
Axelrod doesn't support python 2 anymore so you don't need to force float division, you can have this simply as `current_round / 1000`.